.code-samples.meilisearch.yaml

```
@@ -767,6 +767,20 @@ get_batch_1: |-
   client.get_batch(BATCH_UID)
 get_similar_post_1: |-
   client.index("INDEX_NAME").get_similar_documents({"id": "TARGET_DOCUMENT_ID", "embedder": "default"})
+search_parameter_reference_media_1: |-
+  client.index('movies_fragments').search(
+      "",
+      {
+          "media": {
+              "image_url": "https://image.tmdb.org/t/p/w500/gEU2QniE6E77NI6lCU6MxlNBvIx.jpg"
+          },
+          "hybrid": {
+              "embedder": "voyage",
+              "semanticRatio": 1.0
+          },
+          "limit": 3
+      }
+  )
```

I think a code sample that shows multi-modal (e.g., text + image) search would be welcome:

```suggestion
search_parameter_reference_media_1: |-
  client.index('movies').search(
      "a futuristic movie",
      {
          "media": {
              "textAndPoster": {
                  "text": "a futuristic movie",
                  "image": {
                      "mime": "image/jpeg",
                      "data": "base64EncodedImageData"
                  }
              }
          },
          "hybrid": {
              "embedder": "multimodal",
              "semanticRatio": 1.0
          }
      }
  )
```